bin/common.sh

```diff
@@ -6,49 +6,87 @@ readonly clustersFile="$ENV_DIR/env/clusters.yaml"
 readonly helmfileOutputHide="(^\W+$|skipping|basePath=|Decrypting)"
 readonly helmfileOutputHideTpl="(^[\W^-]+$|skipping|basePath=|Decrypting)"
 readonly replacePathsPattern="s@../env@${ENV_DIR}@g"
+readonly otomiToolsImage="otomi/tools:1.4.10"
 
-get_k8s_version() {
+has_docker=$( docker --version &>/dev/null && echo "1" || echo "0")
+dind=0
+cmd_image=''
+
+function print_error() {
+    echo "Error: " $@ >&2
+}
+
+function _shadow() {
+  local cmd="$1"
+  shift
+  local args="$@"
+  if [[ $has_docker -eq 1 && ("${IN_DOCKER}" != "1" || $dind -eq 1) ]]; then
```

Comment:
The value of `dind` is always 0. I think we can remove this flag.

Reply:
Yeah. Not gonna be used in this context I think.

Reply:
@Dunky13 you probably CAN comment inline here, so please follow us here.

But dind might come later, so please discard.

Reply:
@Morriz your initial idea was to create a `yq` alias. We made a function instead. It shadows `yq` command in the same way. Thanks to that we do not need to change anything else. I am surprised that suddenly you changed your mind.